src/components/Modals/Settings/SettingsList.tsx

```diff
@@ -61,15 +58,15 @@ export const SettingsList = ({ appHistory, ...routeProps }: SettingsListProps) =
     if (clickCount === 4) {
       setClickCount(0)
       settings.close()
-      appHistory.push('/flags')
+      history.push('/flags')
```

Comment:
This is now broken, it was actually the only reason why we were drilling down the outer (app) history, not sure if @shapeshift/operations are still using this? If so, we should remove this along with l.45

Reply:
i don't care if we do `window.location.push` here - this is for developers only, but we absolutely need to keep this functionality

Reply:
Ah, checks out! Will fix.

Reply:
Fixed in https://github.com/shapeshift/web/pull/3309/commits/52588179097cffd9ed81416afc50b2620ebb38c5